akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -43,26 +43,30 @@ private[akka] object TestProbeImpl {
   private case object Stop
 
   private def testActor[M](queue: BlockingDeque[M], terminations: BlockingDeque[Terminated]): Behavior[M] =
-    Behaviors.receive[M] { (context, msg) ⇒
-      msg match {
-        case WatchActor(ref) ⇒
-          context.watch(ref)
-          Behaviors.same
-        case Stop ⇒
-          Behaviors.stopped
-        case other ⇒
-          queue.offerLast(other)
+    Behaviors
+      .receive[M] { (context, msg) =>
```

Comment:
Why this?

Reply:
Let's not allow that format, though it's nice for other things like
```scala
  collection
    .func1(...)
    .func2(...)
    .func3
```

Reply:
I agree, I hope it keeps the line breaks if they are there. I'll try
```
optIn.breakChainOnFirstMethodDot = false
```

Reply:
didn't work can't find the setting for that, we would have to ask

Reply:
Agreed